### DISCUSSION TITLE
Is there any IP camera management application on top of Ant Media Server

### DISCUSSION THREAD

**Discussion by umtcylmz** [ant-media/Ant-Media-Server#5502] [FAQs] [answered]
IP cameras are been widely used by society and I belive managing those cameras can be tricky sometimes. Is there any IP camera management tool that works on top of the Ant Media Server? If so, how can I ask for a demo?

**Comment by Mohit-3196** [comment, marked as answer]
Absolutely, managing IP cameras can indeed pose challenges, but fear not! Ant Media Server has a perfect solution in the form of **CamOS**, a cutting-edge surveillance camera management app. **CamOS** is designed to work seamlessly on top of Ant Media Server, and it's here to transform how you handle your IP cameras.

Introducing CamOS - Your Ultimate IP Camera Management Companion
Imagine a world where managing surveillance cameras becomes effortless, efficient, and secure. That's precisely what CamOS brings to the table. As a remarkable application available on the Ant Media Marketplace, CamOS is your ticket to stress-free IP camera management.

**How to Get a Demo**:
Head over to the **Ant Media Marketplace** at https://antmedia.io/marketplace/ and explore the wonders of CamOS. To request a **demo** and experience the magic firsthand, visit https://antmedia.io/camos-cloud-camera/ and simply follow the instructions to get in touch with the Ant Media team.

I hope it helps.
Thank you.
